core/trino-main/src/test/java/io/trino/execution/TestDropTableTask.java

```diff
@@ -99,8 +101,9 @@ public void testDropTableOnMaterializedViewIfExists()
         QualifiedName viewName = qualifiedName("existing_materialized_view");
         metadata.createMaterializedView(testSession, asQualifiedObjectName(viewName), someMaterializedView(), false, false);
 
-        getFutureValue(executeDropTable(viewName, true));
-        // no exception
```

Comment:
This was added in https://github.com/trinodb/trino/pull/8869
so cc @lukasz-stec @sopel39 @raunaqmorarka

Reply:
cc @losipiuk @martint @hashhar  too

Reply:
should we also do the same change for drop/rename view and materialized view to be consistent?

Reply:
@Riddle4045 please do change the tasks indicated by @lukasz-stec  as well.

Reply:
@findinpath thanks, I will revise the changes.

Reply:
Pls do change `SetPropertiesTask` , `RenameViewTask` as well.